jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -424,4 +484,6 @@ def get_partner_disbursements(
                 ~InvoiceModel.receipts.any(cast(ReceiptModel.receipt_date, Date) >= disbursement_date.date()),
                 DistributionCodeModel.stop_ejv.is_(False) | DistributionCodeModel.stop_ejv.is_(None)
             )
+        sql = str(query.statement.compile(compile_kwargs={"literal_binds": True}))
+        print("SQL:", sql)
```

Comment:
do we need these?

Reply:
no, sorry I'll remove, I print out while testing